src/types/express.locals.ts

```diff
@@ -40,33 +40,33 @@ export type SpcpLocals =
   | { uinFin: string; userInfo: string }
   | { [key: string]: never } // empty object
 
-export interface EmailAutoReplyField {
+export interface EmailRespondentConfirmationField {
   question: string
   answerTemplate: string[]
 }
 
-export interface EmailJsonField {
+export interface EmailDataCollationToolField {
   question: string
   answer: string
 }
 
-export interface EmailFormField {
+export interface EmailAdminDataField {
   question: string
   answer: string
   fieldType: BasicField
   answerTemplate: string[]
 }
```

Comment:
use `type` instead of `interface` for data types

Reply:
Side note as a follow up from our discussion - undefined is not added to the field types here, because undefined is an intermediate state only when the fields are being added to the email data. The fields should otherwise always be defined.